code/src/pages/NotFound.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export const NotFound = () => {
```

Comment:
Again, nice and professional touch but we can't really provoke the site to get this to show. Netlify's own error message seems to take over!

Reply:
if you are using netlify to host your app: create inside the `public` folder the `_redirects` file and save this line in it: `/* /index.html 200`